### DISCUSSION TITLE
Disable dim in f,t mode

### DISCUSSION THREAD

**Discussion by semi710** [folke/flash.nvim#244] [Q&A] [answered]
![image](https://github.com/folke/flash.nvim/assets/60490474/2d79dc23-9c22-47a8-9035-e524472c77dc)
how can i disable this dim colors

**Comment by IndianBoy42** [comment, marked as answer]
`opts.backdrop.highlight` see the docs

**Reply by semi710** [reply]
thanks mate
what i exactly wanted is `modes.char.highlight.backdrop`

**Reply by semi710** [reply]
```lua
opts = {
  modes = {
	char = {
	  highlight = {
	  backdrop = false,
	},
  },
}
```

**Reply by fynnfluegge** [reply]
works for me with `f` and `F`. Anyone know how to disable the dim also for `function() require("flash").jump()` with `s`? I tried to tweak the configuration but did not get it.
